ckanext/iati_generator/tests/test_actions_iati_file_create.py

```diff
@@ -0,0 +1,144 @@
+import pytest
+from ckan.tests import factories
+from ckan import model
+from ckanext.iati_generator.models.iati_files import IATIFile, DEFAULT_NAMESPACE
+from ckanext.iati_generator.models.enums import IATIFileTypes
+
+
+@pytest.fixture
+def setup_data():
```

Comment:
Check how we hadle this setp_data in other projects. We usually return a `types.SimpleNamespace` object. Use it